Mathlib/Topology/MetricSpace/GromovHausdorffRealized.lean

```diff
@@ -115,20 +115,15 @@ private theorem maxVar_bound [CompactSpace X] [Nonempty X] [CompactSpace Y] [Non
 private theorem candidates_symm (fA : f ∈ candidates X Y) : f (x, y) = f (y, x) :=
   fA.1.1.1.2 x y
 
+@[local grind .]
```

Comment:
For only two uses, I'm not sure a `local` attribute is worth the confusion it might cause; if I saw such an attribute, I would wonder why it's getting used everywhere in this file (hint: it's not). Instead, I would pass this lemma to `grind` explicitly.